addons/OPT/GPS/fn_isVehicleVisible.sqf

```diff
@@ -1,16 +1,36 @@
-#include "macros.hpp";
-
-
-params ["_vehicle"];
+/**
+* Prüft, ob ein Fahrzeug für den Spieler auf der Karte sichtbar ist
+* 
+* Autor: Senshi
+*
+* Argumente:
+* 0: <OBJECT> _unit Zu prüfendes Vehicle
+*
+* Rückgabewert:
+* 0: <BOOL> True, wenn sichtbar, sonst false
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+* 
+* API:
+* Nein
+* 
+* Beispiel interner Aufruf:
+* _isVisible = [_unit] call FUNC(isVehicleVisible);
+*
+*/
 
-if (isNull _vehicle) exitWith { false };
-
-// Don't show destroyed
-if (damage _vehicle == 1) exitWith { false };
-// Only show manned
-if (({alive _x} count crew _vehicle) == 0) exitWith { false };
-// Only show manned by own side
-if (side group (crew _vehicle) select 0 != side group CLibPlayer) exitWith { false};
+#include "macros.hpp";
 
+params ["_veh"];
 
-true
+!isNull _veh 
+&& damage _veh < 1 // Don't show destroyed
```

Comment:
Ich bin mir nicht sicher, was die ganzen Skriptbefehle anfangen, wenn `_veh` ein `objNull` sein sollte. Deswegen hatte ich in meinem Vorschlag auch lazy-evaluation mit eingebaut

Reply:
Stimmt, SQF macht ja komische Evals. Furchtbar, wenn man in so vielen Sprachen parallel arbeitet...Ändere es um.

Reply:
Ja ich weiß - das tut mir immernoch in der Seele weh, dass das nicht vernünftig implementiert ist xD